sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -205,6 +211,38 @@ public async Task<HttpResponseMessage> ConvertAsync(HttpRequestMessage input, Ca
             return (T)blob.BlobClient;
         }
 
+        private static ParameterBindingData CreateBindingData(BlobAttribute blobAttribute)
```

Comment:
nit: CreateParameterBindingData

Reply:
There's another `CreateParameterBindingData` in here that actually creates the ParameterBindingData type (but different method definition), so maybe I also call this `ConvertToParameterBindingData` and we'll have these two:

For input binding:

`private static ParameterBindingData ConvertToParameterBindingData(BlobAttribute blobAttribute)`

For output binding:

`private static ParameterBindingData ConvertToParameterBindingData(BlobBaseClient input, BlobTriggerAttribute blobAttribute)`